### PR TITLE
Add ItemVolume to aeon URL

### PR DESCRIPTION
**Pull request by nonword**
By request, adding a new ItemVolume= param to Aeon URLS to hold volume information. (Removing it from CallNumber)

### CODE REVIEW

File: lib/utils/aeon-urls.js [hunk 1]
```diff
@@ -37,7 +26,8 @@ const aeonUrlForItem = async (esItem) => {
 
   const baseUrl = aeonBaseUrl + '/aeon/Aeon.dll?'
 
-  const sierraBib = esItem.item.bibs() && esItem.item.bibs().length ? esItem.item.bibs()[0] : null
+  const sierraItem = esItem.item
```
Comment (charmingduchess) [inline lib/utils/aeon-urls.js:29]: while we're in the business of clearing up this ternary logic, maybe `sierraItem.bibs()?.length` will assist?

File: lib/utils/aeon-urls.js [hunk 2]
```diff
@@ -56,11 +46,12 @@ const aeonUrlForItem = async (esItem) => {
   const locationLabel = locationLabelMap[siteCode]
   const holdingLocationId = firstValue(esItem.holdingLocation())?.id?.split(':').pop()
   const isReCAP = /^rc/.test(holdingLocationId)
+  const itemVolume = sierraItem.fieldTag('v')[0]?.value
```
Comment (charmingduchess) [inline lib/utils/aeon-urls.js:49]: why not use the esItem.shelfMark(true) here?

File: lib/utils/aeon-urls.js (hunk 1 above)
Comment (nonword) [reply lib/utils/aeon-urls.js:29]: Good call

Comment (nonword) [reply lib/utils/aeon-urls.js:29]: Fixed

**Review by charmingduchess — APPROVED** [review]

**Pull request merged**